src/pages/iou/WaypointEditor.js

```diff
@@ -113,47 +130,49 @@ function WaypointEditor({transactionID, route: {params: {iouType = '', waypointI
             onEntryTransitionEnd={() => textInput.current && textInput.current.focus()}
             shouldEnableMaxHeight
         >
-            <HeaderWithBackButton
-                title={translate('distance.waypointEditor')}
-                shouldShowBackButton
-                onBackButtonPress={() => {
-                    Navigation.goBack(ROUTES.getMoneyRequestDistanceTabRoute(iouType));
-                }}
-            />
-            <Form
-                style={[styles.flexGrow1, styles.mh5]}
-                formID={ONYXKEYS.FORMS.WAYPOINT_FORM}
-                enabledWhenOffline
-                validate={validate}
-                onSubmit={onSubmit}
-                shouldValidateOnChange={false}
-                shouldValidateOnBlur={false}
-                submitButtonText={translate('common.save')}
-            >
-                <View>
-                    <AddressSearch
-                        inputID={`waypoint${waypointIndex}`}
-                        ref={(e) => (textInput.current = e)}
-                        hint={!network.isOffline ? translate('distance.errors.selectSuggestedAddress') : ''}
-                        containerStyles={[styles.mt4]}
-                        label={translate('distance.address')}
-                        defaultValue={waypointAddress}
-                        onPress={selectWaypoint}
-                        maxInputLength={CONST.FORM_CHARACTER_LIMIT}
-                        renamedInputKeys={{
-                            address: `waypoint${waypointIndex}`,
-                            city: null,
-                            country: null,
-                            street: null,
-                            street2: null,
-                            zipCode: null,
-                            lat: null,
-                            lng: null,
-                            state: null,
-                        }}
-                    />
-                </View>
-            </Form>
+            <FullPageNotFoundView shouldShow={Number.isNaN(parsedWaypointIndex) || parsedWaypointIndex < 0 || parsedWaypointIndex > waypointCount - 1}>
+                <HeaderWithBackButton
+                    title={translate(wayPointDescriptionKey)}
```

Comment:
I think we can delete the waypointEditor translation entries now that they are unused.